metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -186,7 +188,7 @@ def run(
         obj.echo(
             "Deploying *%s* to Kubeflow Pipelines..." % current.flow_name, bold=True
         )
-        run_pipeline_result = flow.create_run_on_kfp(experiment_name, run_name)
+        run_pipeline_result = flow.create_run_on_kfp(experiment_name, run_name, pipeline_parameters)
```

Comment:
This is passing along all `**kwargs` (pipeline_parameters) to be KFP pipeline parameters.

see
https://github.com/zillow/metaflow/blob/tz/kfp_step/metaflow/plugins/aws/step_functions/step_functions_cli.py#L268-L280

They have a check to ensure that only flow parameters are sent along, see https://github.com/zillow/metaflow/blob/tz/kfp_step/metaflow/plugins/aws/step_functions/step_functions_cli.py#L276-L278

Reply:
Added and verified working.